COMMUNITY.md

```diff
@@ -298,6 +298,26 @@ Portions:
 FRR uses Linux kernel style except where noted below. Code which does not
 comply with these style guidelines will not be accepted.
 
+In addition to following the documented Linux kernel style, developers who
+have made changes to files covered by this style should also run
+tools/checkpatch.sh against their change and note any errors and warnings
+produced. [fixme: how to run checkpatch.sh] This script is also run
+automatically as part of the Continuous Integration testing process for
+every pull request.
+
+In general, a developer should modify the submission to eliminate errors
+and warnings from checkpatch. However, some flexibility is needed for cases
+where human judgement yields better clarity than the script. Accordingly,
+it may be appropriate to ignore some checkpatch.sh warnings or errors per
+discussion among the submitter(s) and reviewer(s) of a change.
+
+There are also some blanket exceptions to checkpatch.sh messages:
+
+    1. lines over 80 characters are allowed for text strings to make it
+       possible to search the code for them.
```

Comment:
Please capitalize "lines" and include a reference to the kernel style guide, FRR github issue or both where this is already explicitly documented

https://github.com/FRRouting/frr/issues/1794
https://www.kernel.org/doc/html/v4.10/process/coding-style.html#breaking-long-lines-and-strings

Reply:
Thanks - added.